airbnb/airbnb/Sources/API/Base/Response.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  Response.swift
+//  Starbucks
+//
+//  Created by seongha shin on 2022/05/10.
+//
+
+import Foundation
+import RxSwift
+
+class Response {
+    let statusCode: Int
+    let data: Data
+    let request: URLRequest?
```

Comment:
Response 객체에서 request 이름을 가지고 있으니 어색한거 같아요. 아래와 같이 작성하는게 어떨까요?

```suggestion
    let urlRequest: URLRequest?
```